templates/base.html

```diff
@@ -9,7 +9,7 @@
         <div id="header">
             <div class="row">
                 <div class="col-md-10">
-                    <h1>Header</h1>
+                    <h1>{% block header_text %}Header{% endblock %}</h1>
```

Comment:
I think eventually we'll have some sweet-ass styling (possibly with a background image?) at the top of the page.